src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IUpgradeContext.cs

```diff
@@ -6,11 +6,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.UpgradeAssistant.Analysis;
 
 namespace Microsoft.DotNet.UpgradeAssistant
 {
     public interface IUpgradeContext : IDisposable
     {
+        int InstanceNumber { get; }
```

Comment:
What's this for?

Reply:
Same. Will be cleaned up in future commits